apiserver/graph/schema/dataprocessing.graphqls

```diff
@@ -180,10 +180,22 @@ type DataProcessConfigChildren {
   enable: String
   zh_name: String
   description: String
+  llm_config: LLMConfig
   preview: [DataProcessConfigpreView]
   file_progress: [DataProcessConfigpreFileProgress]
 }
 
+type LLMConfig {
```

Comment:
should we also update `apiserver/pkg/dataprocessing/dataprocessing.go` file to handle this type?

Reply:
not necessary, this file is just for making HTTP requests.